### PR TITLE
Polish some lessons

### LINKED ISSUE #24: Fix spelling errors in episode licenses

### PR DESCRIPTION
**Pull request by SarahAlidoost**
closes #24 #25 #28 
relates to #22

### CODE REVIEW

File: _episodes/format.md [hunk 1]
```diff
@@ -48,179 +48,16 @@ Tabular data have specific properties that are better supported by the spreadshe
 > In Windows, a file should have at least one extension.
```
Comment (Peter9192) [inline _episodes/format.md:48]: I cannot comment on line 41, but could we use a capital there?

File: _episodes/format.md [hunk 2]
```diff
@@ -251,7 +88,6 @@ Here are some recommended formats for different types:
 > Data Management Plan (DMP) covers how data can be stored, described and reused.
 {: .callout}
 
-
 ## Different formats for different purposes
 
 In the previous sections, we learned that different file formats have different properties.
```
Comment (Peter9192) [inline _episodes/format.md:93]: Line 95: let's go back?

**Review by Peter9192 — APPROVED** [review]
Hi Sarah, 
Thanks for adding these improvements. I spotted to more typo's ;-).
Feel free to merge after addressing those, I don't think I need to check again.

**Comment by SarahAlidoost** [comment]
> Hi Sarah,
> Thanks for adding these improvements. I spotted to more typo's ;-).
> Feel free to merge after addressing those, I don't think I need to check again.

Peter, thank you, I fixed those.

**Pull request merged**